### PR TITLE
Refoctoring code.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**

### CODE REVIEW

File: components/Header/style.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 const styles = StyleSheet.create({
   headerContainer: {
-    width: windowWidth * 0.9,
-    height: windowHeight * 0.05,
-    marginTop: windowHeight * 0.04,
+    // width: windowWidth,
+    // width: windowWidth,
+    // height: windowHeight * 0.1,
+    marginTop: 32,
```
Comment (M7Arman) [inline components/Header/style.js:11]: why did you comment these codes?

File: screens/Profile/profile/ProfileScreen.jsx [hunk 2]
```diff
@@ -40,9 +41,15 @@ const Profile = ({navigation}) => {
 
   const accessCamera = async () => await requestCameraPermission(selectFile);
 
+  const renderSwitchValue = (item) => {
+    if(showHide)  loading ?  <SkeletonPhotos /> : <Photos item={item} />
+    else loading ? <SkeletonPosts /> : <Posts item={item}/>
```
Comment (M7Arman) [inline screens/Profile/profile/ProfileScreen.jsx:46]: not readable code, please use if/else with { }

File: screens/authentication/login/LoginScreen.jsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ import styles from './style';
 const LogInScreen = ({navigation}) => {
   const [keyboardStatus, setKeyboardStatus] = useState(false);
   const [hidePassword, setHidePassword] = useState(true);
-
+
```
Comment (M7Arman) [inline screens/authentication/login/LoginScreen.jsx:21]: ?

File: screens/authentication/login/style.js [hunk 4]
```diff
@@ -10,10 +10,10 @@ const styles = StyleSheet.create({
   },
   root: {
     flex: 1,
-    paddingTop: 20,
+    // paddingTop: 20,
   },
   headerContainer: {
-    width: windowWidth,
+    // width: windowWidth,
```
Comment (M7Arman) [inline screens/authentication/login/style.js:16]: why? why not just remove it?

File: utils/CameraPermissionUtils.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import PermissionsAndroid from 'react-native';
+import {PermissionsAndroid} from 'react-native';
```
Comment (M7Arman) [inline utils/CameraPermissionUtils.js:1]: shouldn't be with spaces like this? 
import { PermissionsAndroid } from 'react-native';

File: components/Header/style.js (hunk 1 above)
Comment (MiqayelChaloyan) [reply components/Header/style.js:11]: It expected to use late, but removed for now.

File: screens/Profile/profile/ProfileScreen.jsx (hunk 2 above)
Comment (MiqayelChaloyan) [reply screens/Profile/profile/ProfileScreen.jsx:46]: Updated.

File: screens/authentication/login/style.js (hunk 4 above)
Comment (MiqayelChaloyan) [reply screens/authentication/login/style.js:16]: No need for this, removed.

File: utils/CameraPermissionUtils.js (hunk 5 above)
Comment (MiqayelChaloyan) [reply utils/CameraPermissionUtils.js:1]: Yes, updated as mentioned.

File: screens/authentication/login/LoginScreen.jsx (hunk 3 above)
Comment (MiqayelChaloyan) [reply screens/authentication/login/LoginScreen.jsx:21]: Removed extra spaces.

**Pull request merged**